pkgs/os-specific/linux/gasket/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "gasket";
-  version = "1.0-18";
+  version = "1.0-18-09385d4";
```

Comment:
According to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming, this should be `1.0-18-unstable-2023-09-05`.

I think.